bazel/go_repositories.bzl

```diff
@@ -69,6 +69,7 @@ def go_repositories():
         importpath = "github.com/census-instrumentation/opencensus-proto",
         sum = "h1:glEXhBS5PSLLv4IXzLA5yPRVX4bilULVyxxbrfOtDAk=",
         version = "v0.2.1",
+        build_extra_args = ["-exclude=src"],
```

Comment:
This file was not touched by gazelle. The new repositories and this `build_extra_args` were needed for the build to work. The `build_extra_args` solution was taken from Sourcegraph - this is needed because there are several targets in this repo that could be meant and excluding the `src` folder leads to the correct one.